client-java/controller/src/main/java/org/evomaster/client/java/controller/contentMatchers/StringCollectionMatcher.java

```diff
@@ -0,0 +1,42 @@
+package org.evomaster.client.java.controller.contentMatchers;
+
+import org.hamcrest.Description;
+import org.hamcrest.TypeSafeMatcher;
+import java.util.Collection;
+
+public class StringCollectionMatcher extends TypeSafeMatcher<Collection<String>> {
+    private final Collection<String> value;
+
+    public StringCollectionMatcher(Collection<String> value) {
+        this.value = value;
+    }
+
+    @Override
+    protected boolean matchesSafely(Collection<String> item) {
+        return item.containsAll(value) && value.containsAll(item);
+    }
+
+    @Override
+    public void describeTo(Description description) {
+        //The point of the matcher is to allow comparisons between int and double that have the same valueE.g. that (int) 0 == (double) 0.0
+        description.appendValue(value);
+    }
+
+    public boolean collectionContains(String item){
+        return value.contains(item);
+    }
+
+    public static boolean collectionsMatch(Collection<String> item1, Collection<String> item2){
```

Comment:
isn't using a name like `item1` confusing for a variable representing a collection?
furthermore, indices start from 0, not 1... :)